CHANGELOG.md

```diff
@@ -86,6 +86,10 @@ Language
 * A new constructor `pattErr : Pattern → ErrorPart` of `ErrorPart` for reflection
   is added.
 
+* The reflection primitive `getType` respects the module context it is
+  invoked from instead of returning the type a name would have outside
+  of all modules.
```

Comment:
Mention the `inContext` trick to get the old behavior.

Reply:
Turns out that that doesn't work, it seems like `instanciateDef` still does its job even if the context is reset. `inferType` has the same behaviour, which means that maybe `inContext` doesn't do what it should?

Reply:
Ok, maybe I was naive about it and going into the top context does not exit from the current module stack.

Maybe the original `getType` could be preserved under the name `getClosedType`?

Also, I wonder whether `getType` is now still in sync with `getDefinition`?  E.g., does `getDefinition` for a function instantiate the clauses to the current module parameters (if applicable)?

Reply:
@jespercockx and I agreed now that we don't want to support this behaviour, since it's not something you could do outside of reflection anyway. I've also changed `getDefinition` to be in sync, which should fix #6200 then.

Reply:
The ability for `inContext` to go all the way to the top level. Jesper changed `inContext` to always stay relative to the context the `TC` computation started.